core/src/main/java/org/ldaptive/asn1/DERPath.java

```diff
@@ -208,7 +208,12 @@ public boolean equals(final Object o)
   @Override
   public int hashCode()
   {
-    return LdapUtils.computeHashCode(HASH_CODE_SEED, nodeStack);
+    int hc = HASH_CODE_SEED;
+    int index = 1;
+    for (Node n : nodeStack) {
+      hc += n.hashCode() * index++;
```

Comment:
I don't see anything necessarily _wrong_ with this given the impl of `Node#hashCode()`, which should produce a unique value given same nodes in different order. That said, the following is more consistent with the textbook answer:

    hc = HASH_CODE_SEED * hc + n.hashCode()

Reply:
Pushed a fix.